src/Stratis.Bitcoin/Configuration/Logging/ILoggerFactory.cs

```diff
@@ -0,0 +1,17 @@
+﻿/*
```

Comment:
This code is commented out, should this be removed?

Reply:
Removed.